src/main/ui/src/components/ParkView/ParkProfile.jsx

```diff
@@ -0,0 +1,80 @@
+import { React, useState, useEffect } from "react";
+import { Container } from "react-bootstrap";
+import "./ParkInfo.css";
+import ProfileHeroImage from "./ProfileHeroImage";
+import ThingsToDoAccordion from "./ThingsToDoAccordion";
+import VisitedParkSwitch from "./VisitedParkSwitch";
+
+export default function ParkInfo() {
+  const [parkInfo, setParkInfo] = useState([]);
+
+  let parkCode = "yell";
+  useEffect(() => {
+    //parkCode should change depending on user choice
+    let url =
+      "https://developer.nps.gov/api/v1/parks?parkCode=" +
+      parkCode +
+      "&api_key=IJ1FjAb4SkwSyXOqFhyqxTM3xg66eDcMQoDjKS16";
+    fetch(url)
+      .then((response) => response.json())
+      .then((json) => {
+        setParkInfo(json.data[0]);
+      })
+      .catch((error) => console.log(error));
+  }, []);
+
+  return (
+    <div className="overall-div">
+      <ProfileHeroImage parkInfo={parkInfo} />
+      <Container className="park-description">
+        <div> {parkInfo.description} </div>
+      </Container>
+      <VisitedParkSwitch />
+
+      <ThingsToDoAccordion parkCode={parkCode} parkInfo={parkInfo} />
```

Comment:
Are we using the accordian? I don't see one. If not please remove this code.

Reply:
Accordion is being used to display activities/hikes in the park